### ISSUE TITLE
2.0.4: ZTS build is broken (PHP 5)

### ISSUE THREAD

**Issue by remicollet** [swoole/swoole-src#989]
```
In file included from /usr/include/php-zts/php/Zend/zend_alloc.h:27:0,
                 from /usr/include/php-zts/php/Zend/zend.h:252,
                 from /usr/include/php-zts/php/main/php.h:35,
                 from /builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/php_swoole.h:22,
                 from /builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c:17:
/builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c: In function 'client_onConnect':
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:167:18: error: 'tsrm_ls' undeclared (first use in this function)
 #define TSRMLS_C tsrm_ls
                  ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:168:21: note: in expansion of macro 'TSRMLS_C'
 #define TSRMLS_CC , TSRMLS_C
                     ^~~~~~~~
/builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c:254:53: note: in expansion of macro 'TSRMLS_CC'
         php_swoole_clear_timer_coro(cli->timeout_id TSRMLS_CC);
                                                     ^~~~~~~~~
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:167:18: note: each undeclared identifier is reported only once for each function it appears in
 #define TSRMLS_C tsrm_ls
                  ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:168:21: note: in expansion of macro 'TSRMLS_C'
 #define TSRMLS_CC , TSRMLS_C
                     ^~~~~~~~
/builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c:254:53: note: in expansion of macro 'TSRMLS_CC'
         php_swoole_clear_timer_coro(cli->timeout_id TSRMLS_CC);
                                                     ^~~~~~~~~
/builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c: In function 'zim_swoole_client_coro_connect':
/builddir/build/BUILD/php-pecl-swoole-2.0.4/ZTS/swoole_client_coro.c:731:34: warning: unused variable 'ccp' [-Wunused-variable]
     swoole_client_coro_property *ccp = swoole_get_property(getThis(), 1);
                                  ^~~

```

**Comment by matyhtf** [comment]
Wait a minute, I will fix the problem ZTS.

**Comment by matyhtf** [comment]
fixed

**Comment by remicollet** [comment]
Sorry, but I don't think current fix is enough (tried with 2.0.5 (from git tag as not available in pecl, rev aaa0303)

```
/builddir/build/BUILD/php-pecl-swoole-2.0.5/ZTS/swoole_redis_coro.c: In function 'swoole_redis_onError':
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:167:18: error: 'tsrm_ls' undeclared (first use in this function)
 #define TSRMLS_C tsrm_ls
                  ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:168:21: note: in expansion of macro 'TSRMLS_C'
 #define TSRMLS_CC , TSRMLS_C
                     ^
/builddir/build/BUILD/php-pecl-swoole-2.0.5/ZTS/swoole_redis_coro.c:3737:107: note: in expansion of macro 'TSRMLS_CC'
  zend_update_property_long(swoole_redis_coro_class_entry_ptr, redis->object, ZEND_STRL("errCode"), c->err TSRMLS_CC);
                                                                                                           ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:167:18: note: each undeclared identifier is reported only once for each function it appears in
 #define TSRMLS_C tsrm_ls
                  ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:168:21: note: in expansion of macro 'TSRMLS_C'
 #define TSRMLS_CC , TSRMLS_C
                     ^
/builddir/build/BUILD/php-pecl-swoole-2.0.5/ZTS/swoole_redis_coro.c:3737:107: note: in expansion of macro 'TSRMLS_CC'
  zend_update_property_long(swoole_redis_coro_class_entry_ptr, redis->object, ZEND_STRL("errCode"), c->err TSRMLS_CC);
                                                                                                           ^
```

**Comment by remicollet** [comment]
see pr  #992
